include/z64.h

```diff
@@ -56,6 +55,7 @@
 #include "z64prerender.h"
 #include "z64save.h"
 #include "z64scene.h"
+#include "z64map.h"
```

Comment:
Any reason to move this include from the other line to here? Previously it was ordered alphabetically